### PR TITLE
Update theme_image.json

### PR DESCRIPTION
**Pull request by a-1evs**
week f themes

### CODE REVIEW

**Review by samsherman5 — CHANGES_REQUESTED** [review]
Change the links by copying the image address by right clicking on the actual picture in the website where you get the picture

File: public/config/theme_image.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
     "Sunday":null,
-    "Monday": "https://i.ibb.co/VMkBZf7/Super-Hero.jpg",
-    "Tuesday": "https://i.ibb.co/fprqrrR/Time-Travel.jpg",
+    "Monday": "https://i.ibb.co/Yby9hHd/Hawaii-Night.jpg",
+    "Tuesday": "https://ibb.co/9qVZXs5",
```
Comment (samsherman5) [inline public/config/theme_image.json]: Wrong type of link

File: public/config/theme_image.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
     "Wednesday":"https://i.ibb.co/xLG9DYd/Wacky-2.jpg",
-    "Thursday":"https://i.ibb.co/cY91xfs/Forest.jpg",
-    "Friday":null,
+    "Thursday":"https://ibb.co/MPJN4FY",
```
Comment (samsherman5) [inline public/config/theme_image.json]: Wrong type of link

**Comment by a-1evs** [comment]
https://i.ibb.co/2jsZ2g4/pride-gradient-1.jpg

File: public/config/theme_image.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
+    "Friday":"https://ibb.co/9qVZXs5",
```
Comment (samsherman5) [inline public/config/theme_image.json]: Wrong type of link

**Review by samsherman5 — DISMISSED** [review]
Change the links by copying the image address by right clicking on the actual picture in the website where you get the picture

**Review by samsherman5 — APPROVED** [review]

**Pull request merged**